doc/getting-started.md

```diff
@@ -271,14 +331,14 @@ kspace_modify	gewald 0.45
 ```
 Please notice that the DeePMD does nothing to the direct space part of the electrostatic interaction, because this part is assumed to be fitted in the DeePMD model (the direct space cut-off is thus the cut-off of the DeePMD model). The splitting parameter `gewald` is modified by the `kspace_modify` command.
 
-## Run path-integral MD with i-PI
+### Run path-integral MD with i-PI
 The i-PI works in a client-server model. The i-PI provides the server for integrating the replica positions of atoms, while the DeePMD-kit provides a client named `dp_ipi` that computes the interactions (including energy, force and virial). The server and client communicates via the Unix domain socket or the Internet socket. The client can be started by
 ```bash
 $ dp_ipi water.json
 ```
 It is noted that multiple instances of the client is allow for computing, in parallel, the interactions of multiple replica of the path-integral MD.
 
-`water.json` is the parameter file for the client `dp_ipi`, and [an example](./examples/ipi/water.json) is provided:
+`water.json` is the parameter file for the client `dp_ipi`, and [an example](https://github.com/tuoping/deepmd-kit/blob/devel/examples/water/ipi/water.json) is provided:
```

Comment:
please avoid hard coding

Reply:
Files in a parent dir can't be linked. I deleted it in the next commit (4.21 15:00).